api/adminmonitor/admin_monitor.go

```diff
@@ -54,7 +54,8 @@ func (m *Monitor) Start() {
 		case <-time.After(m.timeout):
 			initialized, err := m.WasInitialized()
 			if err != nil {
-				log.Fatal().Err(err).Msg("")
+				log.Error().Err(err).Msg("")
```

Comment:
As for the other PR, do you think it's worth adding some context to the error msg? 

e.g. "AdminMonitor failed to determine if Portainer is Initialized" or something?

I do think zerolog adds a file and line number so it may not be necessary now.

Reply:
added contexts.